test/config_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'minitest/autorun'
+require 'repository'
+
+class TestRepoConfig < Minitest::Test
+  def test_path_labels_keys_does_not_end_with_backslash
+    paths = Repository.all.values
+      .select{ |r| r.path_labels? }
+      .map{ |r| r.path_labels.keys }
+      .flatten
+      .select { |file_path| file_path.end_with?('/') }
+    assert_equal [], paths, "remove backslash from the following paths: #{paths.join(', ')}"
```

Comment:
It's a slash (`/`), not a backslash (`\`). I'd also mention it's a trailing slash. Note backslash is also in the test name.